pxr/imaging/hdx/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ pxr_library(hdx
         colorCorrectionTask
         drawTargetTask
         effectsShader
+        exposureScaleTask
```

Comment:
The makefile/package/taskController updates all seem to be "exposureScale", but "linearExposureScale" is included as well.  Note that camera ended up with "GetLinearExposureScale" but not "ExposureScale", so possibly we need to nix all of the plain old "exposureScale" stuff.

I couldn't find the pow(2, exposureScale) anywhere in exposureScaleTask, but if we're going with linearExposureScale that doesn't matter.

Reply:
Apologies - this was apparently in an incomplete state of transition to the new "everything is named linearExposure" changes.

So, anyway, completed the renames of all the *ExposureScale to *LinearExposureScale:

https://github.com/PixarAnimationStudios/OpenUSD/compare/cf0b28ae9427c505b1203f2703a43d2da0d5c71c..671584d67090cbf5c0ff0a6e45948d8e37cea3f3